spinnaker-monitoring-daemon/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile places the python server in /opt/spinnaker-monitoring/bin, and expects
 # config to be mounted in /opt/spinnaker-monitoring/config, which includes the
 # spinnaker-monitoring.yml file.
-FROM python:2.7.13-alpine
+FROM python:2.7.14-alpine3.7
```

Comment:
Do we need to pin -apline3.7, or can we specify -alpine?

Reply:
Unfortunately no, since `-alpine` points to the oldest Alpine version rather than to the newest:

```
/tmp → docker run -it python:2.7.14-alpine cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.4.6
PRETTY_NAME="Alpine Linux v3.4"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
/tmp → docker run -it python:2.7.14-alpine3.7 cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.7.0
PRETTY_NAME="Alpine Linux v3.7"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
```